docs/content/setup/migrations/v11.md

```diff
@@ -186,6 +186,59 @@ Imperative methods have been aligned with the MapLibre GL API as far as possible
   - `easeTo()`
 
 
+
+
+
 ## Changes to `LocationManager` Module
 
-The `LocationManager` module now emits `GeolocationPosition` type, renamed from `Location` type. It also reassembles the web [`GeolocationPosition`](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPosition) much more closely. Previously there was partial support for `heading` and `course`. This has be unified as defined by the spec to `heading` which now gives the direction of travel on both platforms. Retrieving the compass direction is nut supported due to native limitations. 
+### `GeolocationPosition` Type
+
+The `LocationManager` module now emits `GeolocationPosition` type, renamed from `Location` type. It also reassembles the web [`GeolocationPosition`](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPosition) much more closely. Previously there was partial support for `heading` and `course`. This has been unified as defined by the spec to `heading` which now gives the direction of travel on both platforms. Retrieving the compass direction is not supported due to native limitations. 
+
+### Incorporation of `requestAndroidPermissions`
+
+The locationManager now also incorporates the `requestAndroidPermissions` function, the standalone function has been removed:
```

Comment:
Excuse my ignorance, but why is a platform specific permission request required?  Shouldn't this just be requestPermissions?

Reply:
I wondered the same and verified the necessity to call this. If I recollect correctly there will be an error thrown when not requesting those permissions before using location on Android. On iOS it will just ask automatically. And sadly only Android provides those [permission functions in React Native](https://reactnative.dev/docs/permissionsandroid).

I also think, platform parity would be awesome, but that would require some [new native code to make it work on iOS](https://maplibre.org/maplibre-native/ios/latest/documentation/maplibre/mlnlocationmanager/requestwheninuseauthorization/), which I think could be a follow-up.